com.unity.render-pipelines.high-definition/Editor/Lighting/HDLightUI.cs

```diff
@@ -732,91 +743,93 @@ static void DrawEmissionContent(SerializedHDLight serialized, Editor owner)
 
             EditorGUI.BeginChangeCheck(); // For GI we need to detect any change on additional data and call SetLightDirty
 
-            if (lightType != HDLightType.Area)
+            if (!isInPreset)
```

Comment:
If I were you I would put this code in a `CED.Conditional` instead

Reply:
what's the difference?

Reply:
you will have the full flow using the core editor drawers, now we have a mix of them.

Reply:
hum i tried to have a conditional here but how do i access isInPreset since it is not part of the serialized value and it doesnt make sense to have it as a static property (unless it does?))?